Doc/library/fnmatch.rst

```diff
@@ -69,22 +69,24 @@ cache the compiled regex patterns in the following functions: :func:`fnmatch`,
               print(file)
 
 
-.. function:: fnmatchcase(filename, pattern)
+.. function:: fnmatchcase(name, pat)
 
-   Test whether *filename* matches *pattern*, returning :const:`True` or
-   :const:`False`; the comparison is case-sensitive and does not apply
-   :func:`os.path.normcase`.
+   Test whether filename *name* matches pattern *pat*,
```

Comment:
Taken the same wording from `fnmatch` above -- `fnmatchcase` doesn't take PathLike objects, so noting it's a string explicitly I think is useful.
```suggestion
   Test whether the filename string *name* matches the pattern string *pat*,
```

Reply:
Good catch!